src/main/scala/firrtl/passes/RemoveCHIRRTL.scala

```diff
@@ -118,7 +119,7 @@ object RemoveCHIRRTL extends Pass {
         set_write(rws, "wdata", "wmask")
       val mem = DefMemory(s.info, s.name, s.tpe, s.size, 1, if (s.seq) 1 else 0,
                   rds map (_.name), wrs map (_.name), rws map (_.name))
-      Block(mem +: stmts)
+      if (stmts.nonEmpty) Block(mem +: stmts) else EmptyStmt
```

Comment:
I'm actually of the opinion that the empty memory should be propagated then deleted later, my test in #327 shows that zero-port memories are, in general, supported